JavaScript/JavaScriptSDK/Context/User.ts

```diff
@@ -29,12 +34,34 @@ module Microsoft.ApplicationInsights.Context {
          * The store region.
          */
         public storeRegion: string;
-        
+
+        static cookieSeparator: string = '|';
+
+        public setAuthId(id: string, accountId? : string) {
+            this.authId = id;
```

Comment:
I would check whether the type of `accountId` is a `string` or not. Since this function is called from the public API and its JS, we could potentially see non-string values.